src/SFA.DAS.AssessorService.Database/StoredProcedures/OppFinder_Update_StandardSummary.sql

```diff
@@ -168,7 +156,7 @@ BEGIN
 	
 	
 	-- populate the StandardVersionSummary table
-	DELETE FROM StandardVersionSummary WHERE 1=1;
+	TRUNCATE TABLE StandardVersionSummary;
```

Comment:
Will have to check Truncate Table permissions to make sure this works in opportunity finder.